_build_jdk_image.sh

```diff
@@ -17,7 +17,7 @@ function _build_docker_image {
 
 	if [ "${jdk_image_name}" == "jdk11-jdk8" ]
 	then
-		echo "ARG LABEL_ZULU_8_VERSION" >> "${TEMP_DIR}"/Dockerfile
+		echo -e "\nARG LABEL_ZULU_8_VERSION" >> "${TEMP_DIR}"/Dockerfile
```

Comment:
this will put the `ARG LABEL_ZULU_8_VERSION` in a new line at the end of the Dockerfile.